docs/fields.md

```diff
@@ -58,7 +58,7 @@ For example: `username = typesystem.String(max_length=100)`
 * `trim_whitespace` - A boolean indicating if leading/trailing whitespace should be removed on validation. **Default: `True`**
 * `max_length` - A maximum number of characters that valid input stings may contain. **Default: `None`**
 * `min_length` - A minimum number of characters that valid input stings may contain. **Default: `None`**
-* `pattern` - A string to be used as a regex that must match. Eg. `patern="^[A-Za-z]+$"` **Default: `None`**
+* `pattern` - A regular expression that must match. This can be either a string or a compiled regular expression object (which can have flags). E.g. `pattern="^[A-Za-z]+$"` **Default: `None`**
```

Comment:
Let's drop the "(which can have flags)" - Just "This can be either a string or a compiled regular expression object." is fine here.

Reply:
done in 6f604e096bee5075fb36815b551bd8c04bcdbe4b